### DISCUSSION TITLE
What is continuity of prompts in a chat session?

### DISCUSSION THREAD

**Discussion by RobOK2050** [brianpetro/obsidian-smart-connections#209] [Prompts]
Do I have to write "Based on my notes" every time? If I specify a /folder/ do I need to do that every continued prompt in the same chat?

Thanks for this, I am just getting started!

**Comment by hittjwX** [comment]
No, you can point out a specific document. I use maps of content (MOC) to cluster ideas if I want to create an outline, summary, or big idea about something specific. 

If you are having a conversation, then start with the "Based on my notes" but it's not necessary if you are narrowing down a topic based on response.
